aws-api/src/main/java/com/amplifyframework/api/aws/SubscriptionEndpoint.java

```diff
@@ -332,7 +339,7 @@ private String buildConnectionRequestUrl(AuthorizationType authType) throws ApiE
         return new Uri.Builder()
             .scheme("wss")
             .authority(authority)
-            .appendPath(path)
+            .path(path)
```

Comment:
What is this change for?

Reply:
This is to address this item from the PR description:

- Fixed the generated url for websockets, it was doing an .appendPath instead of a .path which yielded urls ending in `/%2Fgraphql` (AppSync's real time endpoint wasn't complaining though)

With this change, generated urls go from `wss://.../%2Fgraphql` to `wss://.../graphql`. This is not strictly needed at the moment because AppSync's real time gateway happens to correctly handle the `/%2Fgraphql`, but still a good idea to fix it.

Also, it is a new Url we are building piece by piece, so "setting" a path instead of "appending" to it makes more sense.